nylas/client/client.py

```diff
@@ -360,9 +360,7 @@ def _get_resources(self, cls, extra=None, **filters):
                 self.api_server, self.client_id, cls.collection_name, postfix
             )
 
-        converted_filters = convert_datetimes_to_timestamps(
-            filters, cls.datetime_filter_attrs
-        )
+        converted_filters = create_request_body(filters, cls.datetime_filter_attrs)
```

Comment:
nit: rename `converted_filters` and `converted_data` to something that reflects the new function name. something like `request_body`